### PR TITLE
[TASK] remove deprecations 10

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "10.0.x-dev"
+            "dev-lwolf-remove-deprecations-10": "10.1.x-dev"
```
Comment (ohader) [inline composer.json:27]: left over from testing?

File: Resources/Public/JavaScript/Application.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ define([
 					},
 					success: function (data, status, xhr) {
 						var message = TYPO3.lang['record_count_message'];
-						message = message.replace('{0}', data);
+						message = message.replace('{0}', data['count']);
```
Comment (ohader) [inline Resources/Public/JavaScript/Application.js:27]: Not required (see server-side comment)

File: Classes/Controller/ModuleController.php [hunk 3]
```diff
@@ -413,10 +416,8 @@ public function countAction(ServerRequestInterface $request)
         $connection = GeneralUtility::makeInstance(ConnectionPool::class)
             ->getConnectionForTable($tablename);
         $count = $connection->count('uid', $tablename, []);
-
-        // Send the response
-        $jsonArray = json_encode($count);
-        return new \TYPO3\CMS\Core\Http\JsonResponse($jsonArray);
+        $array = ['count' => $count];
```
Comment (ohader) [inline Classes/Controller/ModuleController.php:419]: I see... `JsonResponse` in the core does an `empty` check and thus ignores zero `0` counts... (that's a core bug)

```
$response = new \TYPO3\CMS\Core\Http\JsonResponse();
return $response->setPayload((int)$count);
```
... would be a work-around for that, keeping previous client-side implementation in place.

Comment (linawolf) [reply Classes/Controller/ModuleController.php:419]: However like this it is easier for ppl to see what to do if they want to return more then one value

**Pull request merged**